.circleci/config.yml

```diff
@@ -11,27 +11,25 @@ references:
 executors:
   php:
     docker:
-      - image: cimg/php:7.4.5-node
+      - image: cimg/php:7.4-node
 
 commands:
   set-up-packages:
     description: "Installing packages and building"
     steps:
-      - run: composer remove --dev phpunit/phpunit && composer install && npm ci && npm run build
-
-  nodegit-workaround:
-    description: "Ensure nodegit runs without error"
-    steps:
-      - run: sudo apt-get update && sudo apt-get install libkrb5-dev
+      - run: composer update
```

Comment:
This'll ensure that the Composer packages are compatible with the PHP version, whether it be `5.6` or `8.1`